server/src/aiController.js

```diff
@@ -0,0 +1,54 @@
+const OpenAI = require('openai');
+
+const ai = async (request, response) => {
+  const { theme, quantity, gender } = request.body;
+  const openai = new OpenAI({
```

Comment:
You might be interested in writing tests for this if you have time. This is a good example where you can test not only the function, but also how it handles various OpenAI responses. You can utilize mock functions to mimic OpenAI: https://jestjs.io/docs/mock-functions